### PR TITLE
Update common attributes for Foreman 3.3

### PR DESCRIPTION
**Pull request by upadhyeammit**
Cherry-pick into:
None

### CODE REVIEW

File: guides/common/attributes.adoc [hunk 1]
```diff
@@ -1,11 +1,11 @@
 // Document state: "nightly" for master, "stable" for last two releases,
 // "unsupported" for the rest and "satellite" for satellite build
-:DocState: nightly
+:DocState: unsupported
```
Comment (maximiliankolb) [inline guides/common/attributes.adoc:3]:
```suggestion
:DocState: stable
```

**Comment by maximiliankolb** [comment]
It would also be great to adjust the GHA workflow to add 3.3.

**Comment by maximiliankolb** [comment]
> It would also be great to adjust the GHA workflow to add 3.3.

Forget what I said; I just saw your other PR. 👍

**Review by maximiliankolb — APPROVED** [review]
LGTM; "https://yum.theforeman.org/client/3.3/" returning 404 is understandable.

We should merge this now so that we have docs for Foreman 3.3 RC phase, don't we? @upadhyeammit

**Comment by upadhyeammit** [comment]
> LGTM; "https://yum.theforeman.org/client/3.3/" returning 404 is understandable.
> 
> We should merge this now so that we have docs for Foreman 3.3 RC phase, don't we? @upadhyeammit

Yep :)

**Comment by upadhyeammit** [comment]
@maximiliankolb thanks for review, merging this now

**Pull request merged**